index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <script src="js/jquery.min.js"></script>
+  <script>
+  $(document).ready(function() { // эта часть всегда пишется
+    // тут пишем методы
+    $("#div1").mouseenter(function(){
+      $("#div1").animate({
+        height: '+=100px',
+        width: '+=100px'
+      }, 3000);
+          $("#div1").fadeToggle("5000");
```

Comment:
1. If need to apply effects in sequence use chaining (4_chaining.html file in jQuery lesson). It should be `$("#div1").animate(...).fadeToggle(...)`
2. you should pass number 5000, not string "5000" to as fadeToggle argument